app/templates/bulk_resource/upload.html

```diff
@@ -5,8 +5,20 @@
     <div class="ui stackable centered grid container">
         <div class="twelve wide column">
             <h2 class="ui header">
-                CSV Upload
+                Upload Resources by CSV
             </h2>
+            <p>
+              You can easily upload multiple resources at once by writing your
+              resource descriptions to any spreadsheet program, such as
+              Microsoft Excel or Google Sheets, and exporting it as a CSV file,
+              which has a .csv file extension. The details for the file format
```

Comment:
maybe instead of saying "the details for the file format are listed below" we should say something like "you can download a template below" 
thoughts?

Reply:
sure! i'm not too particular about the wording

Reply:
ok cool! do you want me to change this or do you want to do that haha